packages/components/package.json

```diff
@@ -70,46 +72,47 @@
     "date-fns": "^2.30.0",
     "react-animate-height": "^3.2.2",
     "react-focus-on": "^3.9.1",
-    "react-intl": "^6.4.6",
     "react-popper": "^2.3.0",
     "react-textfit": "^1.1.1",
     "uuid": "^9.0.1"
   },
-  "devDependenciesComments": {
-    "esbuild": "It appears that 0.19.x breaks the import of rollup-plugin-esbuild"
-  },
   "devDependencies": {
+    "@babel/plugin-transform-react-pure-annotations": "^7.22.5",
     "@cultureamp/frontend-apis": "^8.0.3",
     "@cultureamp/i18n-react-intl": "^2.1.1",
     "@kaizen/tailwind": "^1.1.0",
     "@rollup/plugin-alias": "^5.0.1",
+    "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-commonjs": "^25.0.5",
     "@rollup/plugin-image": "^3.0.3",
     "@rollup/plugin-json": "^6.0.1",
     "@rollup/plugin-node-resolve": "^15.2.2",
     "@rollup/plugin-typescript": "^11.1.5",
     "@types/react-dom": "^18.2.7",
     "@types/react-textfit": "^1.1.1",
+    "agadoo": "^3.0.0",
     "autoprefixer": "^10.4.16",
+    "babel-plugin-pure-static-props": "^0.2.0",
     "concat-cli": "^4.0.0",
-    "esbuild": "0.18.20",
     "query-string": "^8.1.0",
+    "react-intl": "^6.4.7",
     "rollup": "^3.28.1",
     "rollup-plugin-dts": "^6.0.2",
-    "rollup-plugin-esbuild": "^5.0.0",
     "rollup-plugin-ignore": "^1.0.10",
     "rollup-plugin-node-externals": "^6.1.1",
     "rollup-plugin-postcss": "^4.0.2",
     "sass": "^1.66.1",
     "serialize-query-params": "^2.0.2",
     "svgo": "^3.0.2",
     "ts-jest": "^29.1.1",
+    "tslib": "^2.6.2",
```

Comment:
Move this to a `peerDep`

Reply:
Decided to leave as dep, as this gets bundled into the final dist. Could be revisited if there's an issue.